schemas/com.snowplowanalytics.badrows/format_violation/jsonschema/1-0-0

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a format violation bad row",
+  "self": {
+    "vendor": "com.snowplowanalytics.badrows",
+    "name": "format_violation",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "failure": {
+      "type": "object",
+      "description": "Information regarding the format violation",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "description": "Timestamp at which the failure occurred",
+          "format": "date-time"
+        },
+        "message": {
+          "type": "string",
+          "description": "Failure message associated with the format violation",
+          "maxLength": 256
+        }
+      },
+      "required": [ "timestamp", "message" ],
+      "additionalProperties": false
+    },
+    "payload": {
+      "type": "object",
+      "description": "The collector payload that resulted in a format violation",
+      "properties": {
```

Comment:
should we add more top-level properties, like the ones in https://github.com/snowplow/snowplow/blob/master/2-collectors/thrift-schemas/collector-payload-1/src/main/thrift/collector-payload.thrift ?

Reply:
I think the more we can add the better